baseapp-cloudflare-stream-field/baseapp_cloudflare_stream_field/stream.py

```diff
@@ -61,3 +65,23 @@ def download_video(self, video_uid):
             headers=self.request_headers,
         )
         return res.json()
+
+    def clip_video(self, data={}):
+        res = requests.post(
+            f"{self._api_url}/clip",
+            headers=self.request_headers,
+            json=data,
+        )
+
+        if res.status_code == 400:
+            logging.info(res.text)
+            logging.info(res.status_code)
```

Comment:
or better format this into 1 line, maybe with `logging.error` instead of you wanna track this